src/framework/handlers/anim-clip.js

```diff
@@ -12,8 +12,9 @@ import { ResourceHandler } from './handler.js';
  * @ignore
  */
 class AnimClipHandler extends ResourceHandler {
-    constructor() {
-        super('animclip');
+    constructor(app) {
+        super(app);
+        this.handlerType = 'animclip';
```

Comment:
why not make the super constructor accept two parameters: `app`, and the `handlerType`?
so that this would be `super(app, 'animclip');`

Reply:
I thought that might have caused a constructor type clash but it apparently does not so Ive modified it with this change 👍